source/style-guide/seo-guidelines.txt

```diff
@@ -0,0 +1,84 @@
+.. _seo-guidelines:
+
+=====================================
+Search Engine Optimization Guidelines
+=====================================
+
+.. default-domain:: mongodb
+
+Search Engine Optimization (SEO) considers the actual terms that users 
+enter into serach engines (keywords) and employs best practices to 
+improve traffic to web pages from search engines. The following 
+guidance should help you optimize your documentation pages for search.
+
+Titles
+------
+
+Title pages and subsections according to the following SEO best 
+practices:
+
+- Use a maximum of 70 characters.
+
+- Include target keywords.
+
+- Avoid excessive or irrelevant words (keyword stuffing).
+
+- Use a unique title for every page.
+  
+To learn more, see :ref:`titles-and-headings`.
+
+Alternative Text
+----------------
+
+Screen readers read alternative text aloud so that users can better
+understand an on-page image. Specify alterative text according to the 
+following SEO best practices:
+
+- Use a maximum of 125 characters per image.
+
+- Describe the image with specific language.
```

Comment:
Question:
I wasn't sure what "specific language" represents. Since there isn't another item that explains what it should contain, maybe this should be something like "describe the image with sufficient detail to understand what it shows without seeing it."